frontends/web/src/components/sidebar/sidebar.tsx

```diff
@@ -203,30 +204,38 @@ class Sidebar extends Component<Props> {
                 title={t('accountSummary.title')}
                 onClick={this.handleSidebarItemClick}>
                 <div className={style.single}>
-                  <img draggable={false} src={info} alt={t('sidebar.addAccount')} />
+                  <img draggable={false} src={info} />
                 </div>
                 <span className={style.sidebarLabel}>{t('accountSummary.title')}</span>
               </NavLink>
             </div>
           ) : null }
 
-          {
-            accountsByKeystore.map(keystore => (<React.Fragment key={keystore.keystore.rootFingerprint}>
+          { accountsByKeystore.map(keystore => (
+            <React.Fragment key={keystore.keystore.rootFingerprint}>
               <div className={style.sidebarHeaderContainer}>
                 <span className={style.sidebarHeader} hidden={!keystore.accounts.length}>
-                  {t('sidebar.accounts')} - {keystore.keystore.name} {keystore.keystore.connected ? '- CONNECTED' : null}
+                  {`${t('sidebar.account')} - ${keystore.keystore.name}  `}
+                  {keystore.keystore.connected ? (
+                    <Badge
+                      className={style.activeKeystoreIcon}
+                      icon={props => <USBSuccess {...props} />}
+                      type="success"
+                      title={t('sidebar.keystoreConnected')} />
```

Comment:
Actually we might need this badge in other places (manage accounts), I'll rename the key to something not sidebar specific.